adapters/rubicon/rubicon.go

```diff
@@ -586,7 +586,7 @@ func createImpsToExtMap(imps []openrtb2.Imp) (map[*openrtb2.Imp]rubiconExtImpBid
 func prepareImpsToExtMap(impsToExtMap map[*openrtb2.Imp]rubiconExtImpBidder) map[*openrtb2.Imp]rubiconExtImpBidder {
 	preparedImpsToExtMap := make(map[*openrtb2.Imp]rubiconExtImpBidder)
 	for imp, bidderExt := range impsToExtMap {
-		if bidderExt.Bidder.BidOnMultiformat == false {
+		if !bidderExt.Bidder.BidOnMultiformat {
```

Comment:
I agree with @SyntaxNode's other comments, where I prefer `bidderExt.Bidder.BidOnMultiformat == false` over using the `!` operator.

Reply:
reverting back to the ` == false`. But I do prefer the Go idiomatic boolean check and consistency with the Go convention.  Even in this repository, this will be the only check of this type, while there are many cases of the standard form of checking for the boolean to be `false`:
```bash
$ rg -t go 'if !.* =='
exchange/utils.go
270:	if !defined || binfo.OpenRTB == nil {
960:	if !exists || fpdToApply == nil {

config/experiment.go
62:	if !(cfg.AdCerts.Mode == AdCertsSignerModeOff ||

exchange/events.go
84:	if !ev.isEventAllowed() || pbsBid.BidType == openrtb_ext.BidTypeVideo {
107:	if !ev.isEventAllowed() || pbsBid.BidType == openrtb_ext.BidTypeVideo {

endpoints/openrtb2/auction.go
682:		if !impExtBidderExists || impExtBidder == nil {
730:		if !impExtPrebidBidderExists || impExtPrebidBidder == nil {
1147:	if !hasRootSize && len(banner.Format) == 0 && !isInterstitial {

endpoints/openrtb2/auction_test.go
140:			if !info.IsDir() && filepath.Ext(info.Name()) == jsonFileExtension {

adapters/rubicon/rubicon.go
1062:			if !ok || bidCmpOverride == 0 {

endpoints/info/bidders.go
80:	if !exists || len(v) == 0 {

adapters/decenterads/decenterads.go
37:		if !bidderExtExists || len(bidderExt) == 0 {

adapters/bidscube/bidscube.go
38:		if !bidderExtExists || len(bidderExt) == 0 {
```

```bash
 rg -t go 'if .* == false'
adapters/rubicon/rubicon.go
590:		if bidderExt.Bidder.BidOnMultiformat == false { //nolint: staticcheck
```